docs/guide/groups.md

```diff
@@ -15,9 +15,11 @@ repos:
   - {url: git@gitlab.local:acme/three, dest: three}
 
 groups:
-  default:
```

Comment:
Based on the documentation in https://your-tools.github.io/tsrc/ref/manifest-config/#groups I believe the structure must be even different and requires a `repos` key for the list in every group. E.g.
```
groups:
  default:
    repos: []
  g1:
    repos:
      - one
      - two
   g2:
     repos:
       - three
 ```

Reply:
@hoexter You are absolutely right. I managed to fix a faulty example with another faulty example 🤦. It should be right now. Thanks for your review!